### DISCUSSION TITLE
Vmware player hardware upgrade at build

### DISCUSSION THREAD

**Discussion by ghost** [CosmosOS/Cosmos#2422] [Ideas]
![image](https://user-images.githubusercontent.com/98218366/193393754-76dd0369-d3cd-484b-b1fb-ec2cb13a7b1a.png)
The current dev kit build the image as vmware workstation 6.5/7 hardware. 
An idea should be to upgrade the hardware to 16.2.x at build time.

**Comment by AsertCreator** [comment]
maybe not 16.2.x but 15? 14? 13? some people may have outdated vmware

**Comment by ghost** [comment]
Yeah you're right
